lib/bootsnap/cli.rb

```diff
@@ -117,6 +153,11 @@ def cache_dir=(dir)
       @cache_dir = File.expand_path(File.join(dir, 'bootsnap/compile-cache'))
     end
 
+    def exclude_pattern(pattern)
+      (@exclude_patterns ||= []) << Regexp.new(pattern)
+      self.exclude = Regexp.union(@exclude_patterns)
```

Comment:
This allows several `--exclude` arguments rather than just one. Otherwise it can get complex to join patterns.